dom0/sd-workstation.top

```diff
@@ -9,14 +9,19 @@ base:
     - sd-upgrade-templates
     - sd-dom0-qvm-rpc
     - sd-sys-whonix-vms
+    - sd-log
     - sd-devices
     - sd-gpg
     - sd-proxy
     - sd-viewer
     - sd-app
     - sd-whonix
     - sd-remove-unused-templates
-    - sd-log
+
+  sd-log-buster-template:
```

Comment:
i don't think the order here is important, it depends what the `requires` state

Reply:
Oh, that is something I was wondering.